include/actor.h

```diff
@@ -3,11 +3,14 @@
 
 #include <string>
 
-class Actor {
+#include "locatable.h"
+
+class Actor : public Locatable {
 public:
-    Actor();
+    Actor() = delete;
     ~Actor();
     Actor(std::string);
+    Actor(std::string, const int _x, const int _y);
```

Comment:
Enum type for x and y would make it much more difficult to accidentially set coordinates in wrong order.

Reply:
Issue created for fixing this.